tests/run_bf.sh

```diff
@@ -15,4 +16,9 @@ else
 fi
 export PYTHONPATH
 
-exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
+bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json \
+        -n prplmesh_docker -x test_flows -o ${resultdir}
+
+# if bft returned nonzero return that; else return failed test count
+if [ $rc -ne 0 ]; then exit $rc; fi
```

Comment:
Can't this be simplified to
```suggestion
bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json \
        -n prplmesh_docker -x test_flows -o ${resultdir} \
    || exit $?
```

You definitely are missing a `rc=$?` somewhere. As observed by shellcheck :-)

Reply:
Yeah I need to start using a tool like shellcheck. I changed that part right before pushing it :(